instance/tests/models/test_openedx_database_mixins.py

```diff
@@ -321,12 +321,24 @@ def make_nested_group_info(var_names, databases):
                 database={"name": "edx_notes_api", "user": "notes"},
                 include_port=False
             ),
-            "ECOMMERCE_": make_nested_group_info(
-                ["DEFAULT_DB_NAME", "DATABASES"],
-                [{"name": "ecommerce", "user": "ecommerce", "additional_settings": {
-                    "ATOMIC_REQUESTS": True,
-                    "CONN_MAX_AGE": 60,
-                }}]
+            "ECOMMERCE_": {
+                "vars": ["DATABASES"],
+                "values": [{
+                    "default": {
+                        "ENGINE": 'django.db.backends.mysql',
+                        "NAME": "{{ ECOMMERCE_DATABASE_NAME }}",
+                        "USER": "{{ ECOMMERCE_DATABASE_USER }}",
+                        "PASSWORD": "{{ ECOMMERCE_DATABASE_PASSWORD }}",
+                        "HOST": "{{ ECOMMERCE_DATABASE_HOST }}",
+                        "PORT": expected_port,
```

Comment:
All I wanted to test here was the port value, but I couldn't see a way around doing this other than adding a one-off test for port.

Reply:
Oh yeah.. weird that they didn't add `ECOMMERCE_DATABASE_PORT` upstream too?

Fine to do it this way -- since it also verifies the presence of the `ECOMMERCE_DATABASES` stanza.

Reply:
The new sandboxes based on this change seem to have worked. 
I'll just rebase / squash and update this pull request.